h/services/profile_group.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think we should probably put `from __future__ import unicode_literals` at the top of every file as well, even ones that don't need it, because it's too easy to forget to put it later. (Probably not all files in h have it currently but they should have.)